.travis.yml

```diff
@@ -2,4 +2,15 @@ language: python
 python:
   - 2.7
   - 3.6
-script: pytest
+  - 3.7
+  - 3.8
+  - nightly
+
+matrix:
+  allow_failures:
+    - python: 3.5, nightly
```

Comment:
3.5 is not listed above - do we need this version here?

Reply:
Sorry my bad. Discussed with @ricea , we should at least test on python versions that have python community support (not reaching End-of-life). However since python 3.5 is old, we do not intend to support it in the long run. Hence, we allow tests to fail on python 3.5

Reply:
I find it confusing that "nightly" is listed under both "python:" and "allow_failures:", but "3.5" is only listed until "allow_failures:".